pages/en/lb4/Application.md

```diff
@@ -124,6 +124,42 @@ In the above example:
 - injection calls for `repositories.widget` will be handled by a singleton
 instance of the `WidgetRepository` class.
 
+#### Components
+```ts
+app.component(MyComponent);
+app.component(RestComponent);
+```
+The `component` function allows binding of component constructors within
+your `Application` instance's context.
+
+For more information on how to make use of components,
+see [Using Components](Using-components.html).
+
+#### Controllers
+```ts
+app.controller(FooController);
+app.controller(BarController);
+```
+Much like the component function, the `controller` function allows
+binding of [Controllers](Controllers.html) to the `Application` context.
+
+#### Servers
+```ts
+app.server(RestServer);
+app.servers([MyServer, GrpcServer]);
```

Comment:
Should we eventually get rid of `app.servers()` (the plural version) for consistency? It's out of scope of this pull request though.

Reply:
I think that would be a good idea ... at least till we have plans to re-visit multi-server support.